src/libs/ReportUtils.js

```diff
@@ -1362,6 +1362,7 @@ function getTransactionDetails(transaction) {
     const report = getReport(transaction.reportID);
     return {
         created: TransactionUtils.getCreated(transaction),
+        createdMMDYYYY: TransactionUtils.getCreated(transaction, CONST.DATE.MONTH_DAY_YEAR_FORMAT),
```

Comment:
Could we post process this from `created` where its needed instead of returning it/ computing this in every `getTransactionDetails` we call and it does not need the `createdMMDYYYY `?

Reply:
because that requires more refactoring because of how TransactionUtils.getCreated(transaction), was written originally

Reply:
Do you mean pass in the date format down from getTransactionDetails?

Reply:
I think I could also just remove the refactor and use format() in the file to reformat the created result, but not positive